tomviz/CMakeLists.txt

```diff
@@ -158,10 +158,22 @@ set(python_files
   RandomParticles.py
   TiltSeries.py 
   )
+
+# Make symlinks where possible, file copies where necessary in the build tree.
+if(UNIX)
+  set(script_cmd "create_symlink")
+else()
+  set(script_cmd "copy_if_different")
+endif()
+make_directory("${tomviz_BINARY_DIR}/share/tomviz/scripts")
 foreach(file ${python_files})
-  set(file python/${file})
-  file(COPY ${file} DESTINATION "${tomviz_BINARY_DIR}/share/tomviz/scripts")
-  install(FILES ${file} DESTINATION "${tomviz_data_install_dir}/scripts" COMPONENT "Scripts")
+  list(APPEND SOURCES "python/${file}")
+  execute_process(COMMAND ${CMAKE_COMMAND} -E ${script_cmd}
+    "${tomviz_SOURCE_DIR}/tomviz/python/${file}"
+    "${tomviz_BINARY_DIR}/share/tomviz/scripts/${file}")
+  install(FILES ${file}
```

Comment:
This should be `install(FILES python/${file}`.  Right now `make install` fails since it cannot find the file.